composer.json

```diff
@@ -6,18 +6,18 @@
     "require": {
         "php": "^8.2",
         "ext-amqp": "*",
-        "symfony/amqp-messenger": "^6.0|^7.0",
-        "symfony/http-kernel": "^6.0|^7.0",
-        "symfony/messenger": "^6.0|^7.0"
+        "symfony/amqp-messenger": "^7.2",
+        "symfony/http-kernel": "^7.2",
+        "symfony/messenger": "^7.2"
```

Comment:
min. wersja żeby korzystać z keepalive to 7.2
Chyba można już olać tutaj 6.4 - i tak zaraz wszędzie będziemy na 7